src/simsearch.cc

```diff
@@ -281,8 +281,7 @@ namespace dd
     if (fileops::file_exists(index_filename))
       {
         if (_ondisk)
-          _findex
-              = faiss::read_index(index_filename.c_str(), faiss::IO_FLAG_MMAP);
+          _findex = faiss::read_index(index_filename.c_str());
```

Comment:
log->warn("ondisk not supported, fallback to memory") ?

Reply:
@Bycob that could be a big time saver for someone figuring out why in the future

Reply:
There's no logger inside simsearch at the moment.

Reply:
Maybe we should add one?